### PR TITLE
Enriched the drag end event in goog.fx.AbstractDragDrop 

### PR DESCRIPTION
**Pull request by Znegl**
Enriched the drag end event in goog.fx.AbstractDragDrop with drop target data, to make it possible to distinguish between successful and unsuccessful drag ends.

### CODE REVIEW

File: closure/goog/fx/abstractdragdrop.js [hunk 1]
```diff
@@ -517,7 +517,10 @@ goog.fx.AbstractDragDrop.prototype.endDrag = function(event) {
   }
 
   var dragEndEvent = new goog.fx.DragDropEvent(
-      goog.fx.AbstractDragDrop.EventType.DRAGEND, this, this.dragItem_);
+      goog.fx.AbstractDragDrop.EventType.DRAGEND, this, this.dragItem_,
```
Comment (joeltine) [inline closure/goog/fx/abstractdragdrop.js]: Seems like a decent change. I'd like to see a unit test though.

**Comment by joeltine** [comment]
Ping the thread, if you add tests.

**Comment by Znegl** [comment]
Sorry I haven't reacted the last couple of weeks. Are there any resourcers on how to make unit tests for the library or should i just guess based on the existing tests?
I've tried googling it, but couldn't find anything.

**Comment by joeltine** [comment]
Your best bet is to follow the existing patterns in https://github.com/google/closure-library/blob/master/closure/goog/fx/abstractdragdrop_test.js. Just open the associated `_test.html` file to run the tests.

**Comment by joeltine** [comment]
@Znegl ping the thread if you add a test.

**Comment by Znegl** [comment]
I've just pushed a unit test to the branch form which this pull request is made.

**Comment by joeltine** [comment]
I'm not seeing a test in this PR

**Comment by Znegl** [comment]
I couldn't see it at first either, but it's  there: https://github.com/Znegl/closure-library/commit/171a900e8f68829b74ca8d9bc778c80ffc27e7e6

**Pull request merged**